airbyte-integrations/bases/source-acceptance-test/CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 # Changelog
 
+## 0.1.6
+Add configurable validation of schema for all records in BasicRead test: https://github.com/airbytehq/airbyte/pull/4345
+The validation is ON by default. 
+To disable validation for the source you need to set `validate_schema: off` in the config file.
```

Comment:
Perhaps `validate_schema: False` ?

Reply:
I think this should be on, so we will introduce this aggressively and then fix tests for specific connectors. In this way, we will ensure that new connectors follow best practices from the begining.

Reply:
if you mean that `off` should be `false` instead, then it is pydantic feature to encode boolean as true, 1, on, yes...
https://pydantic-docs.helpmanual.io/usage/types/#booleans